jina/types/request/__init__.py

```diff
@@ -31,7 +31,7 @@
 )
 
 
-class Request(ProtoTypeMixin, DocsPropertyMixin, GroundtruthPropertyMixin):
+class Request(ProtoTypeMixin):
```

Comment:
if you remove these `DocsPropertyMixin, GroundtruthPropertyMixin`, then `Response` lose the type hint